content/en/docs/Installation/_index.md

```diff
@@ -79,6 +79,40 @@ If you'd rather deploy the manifest directly, simply apply the latest release fi
 kubectl create -f https://raw.githubusercontent.com/kyverno/kyverno/main/definitions/release/install.yaml
 ```
 
+## Verify Kyverno image authenticity using cosign
+We sign the official container images that are built on each commit to the main branch. These are signed using cosign. To verify the container image, download our public key into a file named cosign.pub and then:
```

Comment:
Change `We sign the official container images that are built on each commit to the main branch. These are signed using cosign.` to `Kyverno container images are signed using [Cosign](https://github.com/sigstore/cosign)`.

Change `download our public` to `download the organization's public`